hyp3_gamma/rtc/rtc_sentinel.py

```diff
@@ -28,18 +29,38 @@
 from hyp3lib.rtc2color import rtc2color
 from hyp3lib.system import gamma_version
 from hyp3lib.utm2dem import utm2dem
+from hyp3lib import saa_func_lib as saa
+
 from osgeo import gdal, gdalconst, ogr
 
 import hyp3_gamma
 from hyp3_gamma.dem import get_geometry_from_kml, prepare_dem_geotiff
 from hyp3_gamma.rtc.coregistration import CoregistrationError, check_coregistration
 from hyp3_gamma.util import set_pixel_as_point, unzip_granule
 
+
 log = logging.getLogger()
 gdal.UseExceptions()
 ogr.UseExceptions()
 
 
+def createPowerDB(fi, nodata=None):
+    f = saa.open_gdal_file(fi)
+    in_nodata = f.GetRasterBand(1).GetNoDataValue()
+    (x,y,trans,proj,data) = saa.read_gdal_file(f)
+    ch = (data == in_nodata) | (data <= 0.0)
+    data[ch] = np.NaN
+    powerdb = 10*np.log10(data)
+    if not nodata:
+        #nodata = np.iinfo(data.dtype).max
+        nodata = np.NaN
```

Comment:
hyp3lib masks out data using this legacy pattern that is rather brittle to not-numbers (`NaN`, `-inf`, etc):
https://github.com/ASFHyP3/hyp3-lib/blob/develop/hyp3lib/byteSigmaScale.py#L17
 
 So `nodata` will need to be:
 
 ```
 nodata = np.finfo(data.dtype).max.astype(float)
 ```
 (Note the `finfo` not `iinfo`)
 
 The `astype(float)` is required to the `saa.write_gdal_file_float` call later else you'll get a 
```
TypeError: in method 'Band_SetNoDataValue', argument 2 of type 'double'
```